systemd/lcd-session-restart.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Sync LCD time
```

Comment:
Minor nit. This doesn't exclusively synchronize the LCD time. In this PR you also synchronize the display color.